docs/data/joy/components/alert/AlertColors.tsx

```diff
@@ -39,10 +39,12 @@ export default function AlertColors() {
       <Sheet sx={{ pl: 4, ml: 3, borderLeft: '1px solid', borderColor: 'divider' }}>
         <Typography
           level="body-sm"
-          fontWeight="xl"
           id="variant-label"
           textColor="text.primary"
-          mb={1}
+          sx={{
+            fontWeight: 'xl',
+            mb: 1,
+          }}
```

Comment:
Are there ways to fit those into a single line?

```suggestion
          sx={{ fontWeight: 'xl', mb: 1 }}
```

Related to https://github.com/mui/pigment-css/issues/102, so the impact isn't as important as +4K LOCs

<img width="161" alt="SCR-20240601-rqow" src="https://github.com/mui/material-ui/assets/3165635/0ae1c6b6-f912-4fc0-b7e2-592661b9c329">